docs/backend/auth-twitter.md

```diff
@@ -84,7 +85,9 @@ application.
    ![twitter-7](https://user-images.githubusercontent.com/22895284/51078803-9a9a2700-16bb-11e9-8f27-dbfe04b52031.png)
 
 10. From the same dashboard access the **Keys and tokens** and change them
-    accordingly (name of Twitter key -> name of our `ENV` variable):
+    accordingly (name of Twitter key -> name of our `ENV` variable). Be sure to
+    copy the _access token_ and _access token secret_ right away because it is
+    the only time you will see it.
```

Comment:
Maybe we could clarify this: `because it is the only time you will see it.` could be `because it will be hidden from you in the future.`

Thoughts?

Reply:
Sounds good to me, I'll fix it.